nevergrad/benchmark/experiments.py

```diff
@@ -563,7 +563,9 @@ def fastgames(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
     seedg = create_seed_generator(seed)
     algos = ["NaiveTBPSA", "ScrHammersleySearch", "PSO", "OnePlusOne",
              "CMA", "TwoPointsDE", "QrDE", "LhsDE", "Zero", "StupidRandom", "RandomSearch", "HaltonSearch",
-             "RandomScaleRandomSearch", "MiniDE", "SplitOptimizer5", "NGO", "Shiva", "DiagonalCMA"]
+             "RandomScaleRandomSearch", "MiniDE", "SplitOptimizer5", "NGO", "Shiva", "DiagonalCMA",
```

Comment:
Eventually, if they are used for only one experiment, there is no need creating them in experimental variants, you can directly create them for the experiment:

```python
algos += [ConfSplitOptimizer(num_optims=num, progressive=True).set_name(f"ProgOptimizer{num}") for num in [3, 5, 9, 13]]
```

Reply:
Done, thx.

Reply:
Actually the way the code is structured means that we need a str, not an algorithm.

Reply:
What do you mean? The code accept both, the typing may complain but nevermind this.
Also, when I say "eventually" it meant not necessarily now, we'll discuss this later ;)